account.h

```diff
@@ -0,0 +1,108 @@
+#ifndef account_H
+#define account_H
+
+using json = nlohmann::json;
+
+class account {
+protected:
+    int balance{};
+    std::string id;
+    std::string name;
+    std::string surname;
+
+public:
+    int getBalance() {
+        return balance;
+    }
+    std::string getID() {
+        return id;
+    }
+    std::string getName() {
+        return name;
+    }
+    std::string getSurname() {
+        return surname;
+    }
+
+    void CreateAccount(account& a);
+    void UpdateAccount(account& a, const int& id_pos);
+    account(std::string i, std::string n, std::string s, int b);
+};
+
+account::account(std::string i, std::string n, std::string s, int b)
```

Comment:
1) It would be better to use constructor initializer list
2) pass std::string type arguments as const std::string&